src/LuaTransformer.ts

```diff
@@ -3811,12 +3811,12 @@ export class LuaTransformer {
     }
 
     private getImportPath(relativePath: string): string {
-        const rootDir = this.options.rootDir || path.resolve(".");
-        const absoluteImportPath = this.formatPathToLuaPath(this.getAbsoluteImportPath(relativePath));
-        const absoluteRootDirPath = this.formatPathToLuaPath(rootDir);
+        const rootDir = this.options.rootDir && path.resolve(this.options.rootDir) || path.resolve(".");
```

Comment:
I'm not entirely sure what I'm looking at but it looks like a lua ternary conditional? In TS those look like: `this.options.rootDir ? path.resolve(this.options.rootDir) : path.resolve(".")`

Reply:
Been writing too much Lua, I'll turn that into a normal ternary